### PR TITLE
Fix PHP 8.2 notices

### PR DESCRIPTION
**Pull request by truongwp**
Related issue: https://github.com/Strategy11/formidable-pro/issues/4042

### CODE REVIEW

**Review by Crabcyborg — APPROVED** [review]
Thanks @truongwp!

If a deprecation message happens for using `FrmFormsController` with `self` inside of a view, I think Psalm can catch those automatically. We just have that rule turned off right now.

https://github.com/Strategy11/formidable-forms/blob/5cead1d842e4fdee2506308c599521e47a034e0a/psalm.xml#L61

It may be the `InaccessibleMethod` error? Or maybe `InvalidScope`?

File: classes/views/frm-forms/settings-advanced.php [hunk 1]
```diff
@@ -87,7 +87,7 @@
 </p>
 
 <?php
-is_callable( 'self::render_spam_settings' ) && self::render_spam_settings( $values );
+is_callable( 'FrmFormsController::render_spam_settings' ) && FrmFormsController::render_spam_settings( $values );
```
Comment (Crabcyborg) [inline classes/views/frm-forms/settings-advanced.php:90]: I approved this a bit too quickly.

This function is private. Now PHPStan doesn't like it:

```
Error: Call to private static method render_spam_settings() of class FrmFormsController.
 ------ --------------------------------------------------------------- 
  Line   classes/views/frm-forms/settings-advanced.php                  
 ------ --------------------------------------------------------------- 
  90     Call to private static method render_spam_settings() of class  
         FrmFormsController.                                            
 ------ --------------------------------------------------------------- 
```

Maybe we should just make it public.

Comment (truongwp) [reply classes/views/frm-forms/settings-advanced.php:90]: Oh I missed that. Thank you.

**Comment by truongwp** [comment]
@Crabcyborg I tried enabling both of those rules in https://github.com/Strategy11/formidable-forms/pull/1076. It's related to the fix in this PR. This PHP 8.2 notice is for `self` in a callable.

**Pull request merged**